lib/block-supports/background.php

```diff
@@ -74,7 +66,23 @@ function gutenberg_render_background_support( $block_content, $block ) {
 		return $block_content;
 	}
 
-	$styles = gutenberg_get_background_support_styles( $block_attributes['style']['background'] );
+	// Grab any default value from block.json.
+	$background_size_default              = isset( $block_type->supports['background']['backgroundSize'] ) && is_string( $block_type->supports['background']['backgroundSize'] ) ? $block_type->supports['background']['backgroundSize'] : null;
```

Comment:
Wondering if I need to escape block.json values? E.g., `esc_attr( $block_type->supports['background']['backgroundSize'] )`

I can't see a precedent or if they're parsed and escaped in Core...